config/default/manager_metrics_patch.yaml

```diff
@@ -0,0 +1,18 @@
+# This patch exposes metrics endpoint in plain HTTP 8080 port
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
```

Comment:
I understand the namespace should not be set

Reply:
At least, we know it does not have any effect.

The auth proxy patch has the same namespace https://github.com/3scale/3scale-operator/blob/1b7b317fda2d4e0c1022d7e053dfc1ecb6956945/config/default/manager_auth_proxy_patch.yaml#L7

```yaml

apiVersion: apps/v1
kind: Deployment
metadata:
  name: controller-manager
  namespace: system
spec:
```

Reply:
:+1: